app/src/main/java/org/piramalswasthya/sakhi/ui/abha_id_activity/aadhaar_id/find_abha/FindAbhaFragment.kt

```
@@ -205,18 +222,17 @@ class FindAbhaFragment : Fragment() {
                     binding.ivValidBenName.setImageResource(R.drawable.ic_check_circle_grey)
                 }
 
-                isValidBenName = (s != null && s.length >= 3)
+                isValidBenName = (s != null && s.length >= 3 && HelperUtil.isValidName(s.toString()))
                 if (isValidBenName)
                     parentViewModel.setBeneficiaryName(s.toString())
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)